packages/cms/plugin/routes/verify-invite-code.ts

```diff
@@ -0,0 +1,42 @@
+import { route, User } from '@tensei/common'
+import { Request, Response } from 'express'
+
+export const verifyInviteCode = route('Verify Invite Code')
+  .post()
+  .noCsrf()
+  .handle(async (request: Request, response: Response) => {
+
+    const { body, repositories } = request
+
+    try {
+      await request.config.indicative.validator.validateAll(
+        body,
+        {
+          inviteCode: 'required'
+        },
+        {
+          'inviteCode.required': 'The invite code is required.'
+        }
+      )
+    } catch (error) {
+      return response.status(422).json({
+        errors: error
+      })
+    }
+
+    // check that email does not exist
+    const invitedMember: User = await repositories.adminUsers().findOne({ inviteCode: body.inviteCode })
+
+    if (!!invitedMember?.email == false) {
```

Comment:
For simplicity  I think we'd just check `if (invitedMember)`. Also its always safe to do boolean checks with `===`.